plugins/woocommerce/includes/class-wc-order-item-shipping.php

```diff
@@ -153,17 +153,6 @@ public function set_taxes( $raw_tax_data ) {
 		}
 	}
 
-	/**
-	 * Set tax_status.
-	 *
-	 * @param string $value Tax status.
-	 */
-	public function set_tax_status( $value ) {
-		if ( in_array( $value, array( 'taxable', 'none' ), true ) ) {
-			$this->set_prop( 'tax_status', $value );
-		}
-	}
-
```

Comment:
This was introduced in https://github.com/woocommerce/woocommerce/pull/46069 which is on 9.6 milestone. Even if shortlived, if this PR lands on 9.7, any code attempting to use it will throw a fatal (it doesn't exist in upstream extends or traits). Can't we ship the setter anyway?

Reply:
The setter has no value because the value is not read. I would not worry too much about this tbh.

Reply:
It's still considered public code and we should be proactive about preventing that fatal, no matter how low the chance.